pkg/provider/resource_cluster.go

```diff
@@ -264,6 +271,10 @@ func (c *ClusterResource) Schema() *schema.Resource {
 func (c *ClusterResource) Create(ctx context.Context, d *schema.ResourceData, meta any) diag.Diagnostics {
 	client := api.BuildAPI(meta).ClusterClient()
 
+	err := d.Set("cluster_type", "cluster")
+	if err != nil {
+		return nil
```

Comment:
If there is an error, why are we returning nil? 🤔

Reply:
missed this, fixed it.